benchmark/src/main/scala/org/typelevel/catbird/benchmark/RerunnableBenchmark.scala

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2022 Typelevel
```

Comment:
I'm not sure this is a valid copyright header for this repo. We probably shouldn't merge this until we get that sorted out.